Common/Tools/trackTuner.h

```diff
@@ -0,0 +1,679 @@
+  float oneOverPtCurrent = 0.;        // 1/pt old
+  float oneOverPtUpgr = 0.;           // 1/pt new
+  ///////////////////////////////
+
+  o2::ccdb::CcdbApi ccdbApi;
+  std::map<std::string, std::string> metadata;
+
+  std::unique_ptr<TGraphErrors> grDcaXYResVsPtPionCurrent;
+  std::unique_ptr<TGraphErrors> grDcaXYResVsPtPionUpgr;
+
+  std::unique_ptr<TGraphErrors> grDcaZResVsPtPionCurrent;
+  std::unique_ptr<TGraphErrors> grDcaZResVsPtPionUpgr;
+
+  std::unique_ptr<TGraphErrors> grDcaXYMeanVsPtPionCurrent;
+  std::unique_ptr<TGraphErrors> grDcaXYMeanVsPtPionUpgr;
+
+  std::unique_ptr<TGraphErrors> grDcaZMeanVsPtPionCurrent;
+  std::unique_ptr<TGraphErrors> grDcaZMeanVsPtPionUpgr;
+
+  std::unique_ptr<TGraphErrors> grOneOverPtPionCurrent;
+  std::unique_ptr<TGraphErrors> grOneOverPtPionUpgr;
+
+  std::unique_ptr<TGraphErrors> grDcaXYPullVsPtPionCurrent;
+  std::unique_ptr<TGraphErrors> grDcaXYPullVsPtPionUpgr;
+
+  std::unique_ptr<TGraphErrors> grDcaZPullVsPtPionCurrent;
+  std::unique_ptr<TGraphErrors> grDcaZPullVsPtPionUpgr;
+
+  /// @brief Function to configure the TrackTuner parameters
+  /// @param inputString Input string with all parameter configuration. Format: <name>=<value>|<name>=<value>
+  /// @return String with the values of all parameters after configurations are listed, to cross check that everything worked well
+  std::string configParams(std::string inputString)
+  {
+
+    std::string delimiter = "|";
+    std::string assignmentSymbol = "=";
+
+    LOG(info) << "[TrackTuner] === ";
+    LOG(info) << "[TrackTuner] === Parameter configuration via std::string";
+    LOG(info) << "[TrackTuner] === Required format: \"<name>" << assignmentSymbol << "<value>" << delimiter << "<name>" << assignmentSymbol << "<value>" << delimiter << "<name>" << assignmentSymbol << "<value>\"";
+    LOG(info) << "[TrackTuner] === Delimiter symbol: \"" << delimiter << "\"";
+    LOG(info) << "[TrackTuner] === Assignment symbol: \"" << assignmentSymbol << "\"";
+    LOG(info) << "[TrackTuner] === ";
+    LOG(info) << "[TrackTuner]";
+    LOG(info) << "[TrackTuner] >>> Original input string = \"" << inputString << "\"";
+
+    /// Check the format of the input string
+    if (inputString.find(delimiter) == std::string::npos) {
+      // wrong delimiter symbol used
+      LOG(fatal) << "ERROR: delimiter symbol \"" << delimiter << "\" not found in the configuration string. Fix it!";
+    }
+    if (inputString.find(assignmentSymbol) == std::string::npos) {
+      // wrong assignment symbol used
+      LOG(fatal) << "ERROR: assignment symbol \"" << assignmentSymbol << "\" not found in the configuration string. Fix it!";
+    }
+    int spaces = std::count(inputString.begin(), inputString.end(), ' ');
+    if (spaces > 0) {
+      // white spaces to be removed
+      LOG(fatal) << "ERROR: " << spaces << " white spaces found in the configuration string. Remove them!";
```

Comment:
```suggestion
      LOG(fatal) << "delimiter symbol \"" << delimiter << "\" not found in the configuration string. Fix it!";
    }
    if (inputString.find(assignmentSymbol) == std::string::npos) {
      // wrong assignment symbol used
      LOG(fatal) << "assignment symbol \"" << assignmentSymbol << "\" not found in the configuration string. Fix it!";
    }
    int spaces = std::count(inputString.begin(), inputString.end(), ' ');
    if (spaces > 0) {
      // white spaces to be removed
      LOG(fatal) << spaces << " white spaces found in the configuration string. Remove them!";
```
Do not duplicate logging keywords